elastic/integrations
# [Auditd Manager] Add Session Data option

## Proposed commit message

- Updating Auditd Manager manifest to include Session Data option starting from Kibana version 8.16.0.
- Updated logic in the `hbs` file to append Session Data Audit Rules and Session Data Processors when Session Data is selected.
- Updated documentation to include both manual and Toggle Switch options

## Checklist




## How to test this PR locally

To test these changes, go to Kibana -> Integrations -> Create new integration -> upload it as a .zip; and upload the following package:

[auditd_manager-1.18.0.zip](https://github.com/user-attachments/files/17333356/auditd_manager-1.18.0.zip)


**Please note**. This Kibana [PR](https://github.com/elastic/kibana/pull/195159) should first be merged to test this integration.

## Related issues

- Closes https://github.com/elastic/security-team/issues/10330

## Screenshots

Session Data Switcher

<img width="1657" alt="image" src="https://github.com/user-attachments/assets/4a6480f0-27a6-4d6f-9528-a9c43b1e7553">

elastic-agent .yml

<img width="792" alt="image" src="https://github.com/user-attachments/assets/0334a4ce-b5df-4059-a3aa-a0e91e0d47ce">


Docs

<img width="1652" alt="image" src="https://github.com/user-attachments/assets/71e556cd-6e8f-4b3a-9666-0041bb56c59b">
